core/geometry/src/Geometry.ts

```diff
@@ -924,5 +924,14 @@ return false;
  */
 export interface Cloneable<T> {
   /** required method to return a deep clone. */
-  clone (): T | undefined;
+  clone(): T | undefined;
+}
+
+/**
+ * interface for Vector2d/3d isPerpendicularTo and isParallelTo functions
+ * @public
+ */
+export interface PerpParallelOptions {
+  radianSquaredTol?: number;
```

Comment:
Document what the two properties are used for and their default values.

Reply:
done

Reply:
adjusted.

Reply:
I did run `extract-api` before pushing. are you sure I need to re-run?

Reply:
I made changes, adding documentation to undocumented fields. That will change the api-extractor output.

Reply:
I need to edit your comment though. I will make the change and run `extract-api` and push the new change